SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -349,6 +341,8 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
         //      `provider:performAnswerCallAction:` per the WWDC examples.
         // TwilioVoice.configureAudioSession()
         
+        assert(action.callUUID == self.callInvite?.uuid)
```

Comment:
Would move into `self. self.performAnswerVoiceCall()`, otherwise we don't need the `uuid` argument.